jenkins/job-configs/kubernetes-jenkins-pull/kubernetes-pull.yaml

```diff
@@ -0,0 +1,200 @@
+- job:
+    name: kubernetes-pull-test-unit-integration
+    logrotate:
+        daysToKeep: 7
+    properties:
+        - github:
+            url: https://github.com/kubernetes/kubernetes
+        - throttle:
+            max-per-node: 2
+            option: project
+        - raw:
+            xml: |
+                <com.cloudbees.plugins.JobPrerequisites plugin="slave-prerequisites@1.0">
+                    <script>docker version gcloud version</script>
+                    <interpreter>shell script</interpreter>
+                </com.cloudbees.plugins.JobPrerequisites>
+    parameters:
+        - string:
+            name: sha1
+            description: 'sha1 or refname (e.g. origin/pr/N/head) to build'
+    scm:
+        - git:
+            remotes:
+                - remote:
+                    url: https://github.com/kubernetes/kubernetes
+                    refspec: '+refs/heads/*:refs/remotes/upstream/*'
+                - remote:
+                    url: https://github.com/kubernetes/kubernetes
+                    refspec: '+refs/pull/${ghprbPullId}/merge:refs/remotes/origin/pr/${ghprbPullId}/merge'
+            branches:
+                - 'origin/pr/${ghprbPullId}/merge'
+            browser: githubweb
+            browser-url: http://github.com/kubernetes/kubernetes
+            timeout: 20
+            clean:
+                after: true
+            wipe-workspace: false
+            skip-tag: true
+    triggers:
+        - github-pull-request:
```

Comment:
it took around a week from gerrit request to merge, so not too bad.

I'd argue for planning to merge this as-is and then updating after JJB is updated.